script/install.sh

```diff
@@ -5,6 +5,8 @@
 # Authors: Martin Nowak
 # Documentation: https://dlang.org/install.html
 
+SCRIPT_PATH=${BASH_SOURCE[0]}
```

Comment:
Let's better use the full expression as given and explained here: https://stackoverflow.com/questions/59895/how-to-get-the-source-directory-of-a-bash-script-from-within-the-script-itself

tl:dr: it matters for a few special cases.

Reply:
We should first agree on what should happen if the script is called through a symlink. Should compilers be installed alongside the original (then the full expression is needed) or alongside the symlink (works as is)? I think the latter is desirable. You might want to have your installer git repository in `~/projects/installer`, and test it in `~/dlangtest/`. That is possible with `ln -s ~/projects/installer/script/install.sh ~/dlangtest/install.sh` and calling it through `~/dlangtest/install.sh`.

Reply:
Clarified in comment.